bindings/rascal/models/genericmd.py

```diff
@@ -0,0 +1,130 @@
+    ):
+        """Initialize a model and structure template
+
+        Parameters
+        ----------
+        model_json  Filename for a JSON file defining the potential
+        is_periodic Specify whether the simulation is periodic or not
+                    This helps avoid confusion if a geometry's "periodic"
+                    flags have been set improperly, which can happen e.g.
+                    if ASE cannot read the cell information in a file.  If
+                    using a structure template and this is set to True,
+                    will raise an error unless at least one of the PBC
+                    flags in the structure template is on.  If set to
+                    False, will raise an error if all PBC flags are not
+                    off.  Set to None to skip PBC checking.  If not using a
+                    structure template, this setting will determine the PBC
+                    flags of the created atomic structure.
+        structure_template
+                    Filename for an ASE-compatible Atoms object, used
+                    only to initialize atom types and numbers
+        atomic_numbers
+                    List of atom types (atomic numbers) to initialize
+                    the atomic structure in case no structure template
+                    is given
+        """
+        super(GenericMDCalculator, self).__init__()
+        self.model_filename = model_json
+        self.model = load_obj(model_json)
+        self.representation = self.model.get_representation_calculator()
+        self.manager = None
+        # Structure initialization
+        self.is_periodic = is_periodic
+        if structure_template is not None:
+            self.template_filename = structure_template
+            self.atoms = ase.io.read(structure_template, 0)
+            if (is_periodic is not None) and (
+                is_periodic != np.any(self.atoms.get_pbc())
+            ):
+                raise ValueError(
+                    "Structure template PBC flags: "
+                    + str(self.atoms.get_pbc())
+                    + " incompatible with 'is_periodic' setting"
+                )
+        elif atomic_numbers is not None:
+            self.atoms = ase.Atoms(numbers=atomic_numbers, pbc=is_periodic)
+        else:
+            raise ValueError(
+                "Must specify one of 'structure_template' or 'atomic_numbers'"
+            )
+
+    def calculate(self, positions, cell_matrix):
+        """Calculate energies and forces from position/cell update
+
+        positions   Atomic positions (Nx3 matrix)
+        cell_matrix Unit cell (in ASE format, cell vectors as rows)
+                    (set to zero for non-periodic simulations)
+
+        The units of positions and cell are determined by the model JSON
+        file; for now, only Å is supported.  Energies, forces, and
+        stresses are returned in the same units (eV and Å supported).
```

Comment:
This Å is the non ascii character I think. The error is a Python 2 one, which can be fixed by switching to Python 3 or adding `# -*- coding: UTF-8 -*-` as the first line of this file

Reply:
...ok, but that was present in the last working version (3281e07) too. And why would it be using Python 2 to compile the docs? I thought we were pretty clear that this project is Python 3 only.

Reply:
The most maddening thing is that Sphinx won't even tell me which file has the offending characters, so I'm wondering if `nbsphinx` has something to do with this.